executor/src/witgen/machines/fixed_lookup_machine.rs

```diff
@@ -180,7 +181,7 @@ impl FixedLookup {
             .iter()
```

Comment:
I think now we shouldn't skip anymore but check it instead.
Previously we essentially had this so that the lookup is not performed again once we solved it.

Reply:
Done!